src/components/fields/field-select/field-select.component.tsx

```diff
@@ -65,7 +68,7 @@ const SelectField: FC<Props> = ({
         onChange({
           uri: null,
           prefLabel: {
-            no: null
+            nb: null
```

Comment:
Har det noe å si for de som allerede er lagret med 'no'?

Reply:
Nei, det skal gå fint. Både nb og no er prøvd for oversettelser